README.md

```diff
@@ -22,24 +22,33 @@ Version 1.5.1 contains breaking changes and requires re-formating of the add-on
 #### Restoring Configuration after upgrading to 1.5.1
 By default, when upgrading to v1.5.1, the add-on will create a backup of your configuration.yml within your data path: `$DATA_PATH/configuration.yaml.bk`. When upgrading, you should use this to fill in the relevant values into your new config, particularly the network key, to avoid breaking your network and having to repair all of your devices.
 
-
 ## Installation
 
-Add the repository URL via the Hassio Add-on Store Tab: `https://github.com/danielwelch/hassio-zigbee2mqtt`
+Add the repository URL via the Hassio Add-on Store Tab:
+
+    https://github.com/danielwelch/hassio-zigbee2mqtt
 
-The repository includes two add-ons: **zigbee2mqtt** and **zigbee2mqtt-edge**. For a stable release that tracks the released versions of zigbee2mqtt, install zigbee2mqtt. zigbee2mqtt-edge tracks the dev branch of zigbee2mqtt, so you can install the edge version if there are features or fixes in the dev branch that are not yet released.
+The repository includes two add-ons:
+
+- **zigbee2mqtt** is a stable release that tracks the released versions of zigbee2mqtt.
+- **zigbee2mqtt-edge** tracks the `dev` branch of zigbee2mqtt, so you can install the edge version if there are features or fixes in the dev branch that are not yet released.
 
 ## Configuration
 
-Configure the add-on via the Hass.io front-end. The configuration closely mirrors that of `zigbee2mqtt` itself, with a couple of key differences:
+Configure the add-on via the Hass.io front-end under "Supervisor (Hass.io) → Dashboard → zigbee2mqtt".
+
+The configuration closely mirrors that of [zigbee2mqtt itself](https://www.zigbee2mqtt.io/information/configuration.html), with a couple of key differences:
+
 1. Hass.io requires add-on configuration in JSON format, rather than YAML. If you don't understand the difference, you can use a YAML-to-JSON converter.
-2. An additional top-level `data-path` option is required. Set this to the path where you would like the add-on to persist data. Defaults to `/share/zigbee2mqtt`. Note that both `config` and `share` directories are mapped into the container (read-write) and are available to you.
+
+2. An additional top-level `data_path` option is required which defaults to `/share/zigbee2mqtt`. This is the path where the add-on should persist the data. The path must be relative to the Home Assistant shared data directory (which is `/usr/share/hassio` for Hass.io). Note that both `config` and `share` directories are mapped into the container (read-write) and are available to you.
```

Comment:
This is the core change -- we're changing `data-path` to `data_path` and adding a the note about the file path being relative to the HA's shared directory.

As a first time user I tried setting it to my existing zigbee2mqtt configuration location `/home/pi/zigbee2mqtt` which obviously didn't work.